stdlib/InteractiveUtils/test/runtests.jl

```diff
@@ -362,25 +362,27 @@ let err = tempname(),
     finally
         redirect_stderr(old_stderr)
         close(new_stderr)
-        let errstr = read(err, String)
+        let errstr = read(errf, String)
             @test startswith(errstr, """start
                 end
                 Internal error: encountered unexpected error during compilation of f_broken_code:
                 ErrorException(\"unsupported or misplaced expression \\\"invalid\\\" in function f_broken_code\")
                 """) || errstr
             @test !endswith(errstr, "\nend\n") || errstr
         end
-        rm(err)
+        rm(errf)
     end
 end
 end
 
 # Issue #33163
 A33163(x; y) = x + y
 B33163(x) = x
-@test (@code_typed A33163(1, y=2))[1].inferred
-@test !(@code_typed optimize=false A33163(1, y=2))[1].inferred
-@test !(@code_typed optimize=false B33163(1))[1].inferred
+let
+    (@code_typed A33163(1, y=2))[1]
```

Comment:
Can you add some form of `@test` here, both because that catches exceptions and counts this as a test, and because it signals to future readers that we didn't forget the test?

Reply:
I'm just waiting for https://github.com/JuliaLang/julia/pull/51982 to come to some sort of conclusion and will add whatever form gets decided on there.